third_party/nvfuser/csrc/lower_expr_sort.cpp

```diff
@@ -1038,8 +1038,7 @@ void ExprSegmentationSorter::initializeForLoopDependencies() {
       }
 
       // Loops after tv_id are dependent on tv_id
-      dependencies.emplace(GpuLower::current()->caMap()->getConcreteMappedID(
-          tv_id, IdMappingMode::LOOP));
+      dependencies.emplace(concrete_id);
```

Comment:
Why is this changed?

Reply:
I think we have already get the concrete_id in line 1029 , does not need to get again.
auto concrete_id = GpuLower::current()->caMap()->getConcreteMappedID(tv_id, IdMappingMode::LOOP);